### PR TITLE
Add boto script to add go-agent ingress into apps.

### PR DESCRIPTION
**Pull request by doctoryes**
@edx/pipeline-team Please review. 

I removed specific IDs.

### CODE REVIEW

File: admin/boto_to_add_ingress.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+
+LOG_FILENAME = 'security_group_change.txt'
+logging.basicConfig(filename=LOG_FILENAME,level=logging.INFO)
+
+# Info for the security group used by the go-agents.
+GO_AGENT_SECURITY_GROUP_NAME = u'prod-tools-goagent-sg'
+# The values below are fake - replace them with the real values.
+GO_AGENT_SECURITY_GROUP_OWNER_ID = 111111111111
+GO_AGENT_SECURITY_GROUP_ID = u'sg-beefbeef'
```
Comment (macdiesel) [inline admin/boto_to_add_ingress.py]: 8xbadf00d =)

**Comment by macdiesel** [comment]
👍

**Comment by doctoryes** [comment]
👍

**Comment by feanil** [comment]
Made one last change to deal with the rule already existing.

**Pull request merged**